lib/project_types/extension/tasks/get_extensions.rb

```diff
@@ -5,7 +5,10 @@ module Extension
   module Tasks
     class GetExtensions < ShopifyCLI::Task
       def call(context:, type:)
-        organizations = ShopifyCLI::PartnersAPI::Organizations.fetch_with_extensions(context, type)
+        org_id = ShopifyCLI::DB.get(:organization_id)
+        return [] unless org_id
+
+        organizations = ShopifyCLI::PartnersAPI::Organizations.fetch_with_extensions(context, type, id: org_id)
         extensions_from_organizations(organizations, context: context)
```

Comment:
While testing `extension connect`, I've found that it's raising an ugly error when there are no extensions, because of this piece of code from line 27:
```
app["extensionRegistrations"].map do |registration| 
```

It's throwing this error: `NoMethodError: undefined method 'map' for nil:NilClass`.

I know it's not related to your changes, but would you mind fixing it? It should be enough if we do something like this instead:
```
registrations = app["extensionRegistrations"] || []
registrations.map do |registration|
```